packages/lab/src/stepped-tracker/stepReducer.spec.tsx

```diff
@@ -0,0 +1,160 @@
+import { describe, expect, it } from "vitest";
+
```

Comment:
Is there a convention in place already to put tests be in `packages/lab/__tests__` ?

Reply:
Done!

P.S. Personal opinion units tests should be placed close to source code